### PR TITLE
gwastools: manhattanplot

### PR DESCRIPTION
**Pull request by astrovsky01**
FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] - License permits unrestricted use (educational + commercial)
* [x] - This PR adds a new tool or tool collection
* [ ] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

File: tools/gwastoos/manhattan_plot.xml [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+        #end if
+        #if $thinselect.thinning == 'yes':
+            --thin ${thinselect.thin}
+            #if $thinselect.ppb:
+                --ppb ${thinselect.ppb}
+            #end if
+        #end if
+    ]]></command>
+    <inputs>
+        <param name="file" type="data" format="tsv, tabular" label="GWAS data file"/>
```
Comment (bgruening) [inline tools/gwastoos/manhattan_plot.xml]: remove space in formats

File: tools/gwastoos/manhattan_plot.xml [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+        <param name="chromosome" type="integer" value="2" label="Chromosome column number"/>
+        <param name="pvals" type="integer" value="4" label="P-values column number"/>
+        <param name="ymax" type = "float" optional="true" label="Y max"/>
+        <param name="ymin" type = "float" optional="true" label="Y min"/>
+        <param name="trunc" type="boolean" truevalue="--trunc" falsevalue="" label="Show truncation lines?"/>
+        <param name="sig" type="float" optional="true" label="Genome-wide significance level for plotting horizontal line." help="If not specified, no line will be drawn."/>
+        <conditional name="thinselect">
+            <param name="thinning" type="boolean" truevalue="yes" falsevalue="no" label='Specify thinning value?'/>
```
Comment (bgruening) [inline tools/gwastoos/manhattan_plot.xml]: do not use a boolean in conditionals, but rather a select ...

File: tools/gwastoos/.shed.yml [hunk 3]
```diff
@@ -0,0 +1,14 @@
+categories:
+    - Visualization
+    - Variant Analysis
+homepage_url: https://bioconductor.org/packages/release/bioc/html/GWASTools.html
+auto_tool_repositories:
+    name_template: "{{ tool_id }}"
+    description_template: "{{ tool_name }} tool from the ggplot package"
+suite:
+    name: suite_gwastools
```
Comment (bgruening) [inline tools/gwastoos/.shed.yml]: Is that only the first tool from many to come?

File: tools/gwastoos/manhattan_plot.xml [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+            <conditional name="thinselect">
+                <param name="thinning" value="yes"/>
+                <param name="thin" value="1"/>
+                <param name="ppb" value="1"/>
+            </conditional>
+            <output name="manhattanout" ftype="pdf" value="testgraph.pdf" compare="sim_size"/>
+        </test>
+    </tests>
+    <help><![CDATA[
+        Generates a manhattan plot for GWAS data
```
Comment (bgruening) [inline tools/gwastoos/manhattan_plot.xml]: its completely unclear how the input file should look like

File: tools/gwastoos/manhattan_plot.xml [hunk 5]
```diff
@@ -0,0 +1,73 @@
+<tool id="manhattan.plot" name="Manhattan plot" version="0.1.0" python_template_version="3.5">
+    <description>Generate plots for use with GWAS pipelines</description>
+    <requirements>
+        <requirement type="package" version="3.6">r-base</requirement>
+        <requirement type="package" version="1.30.0">bioconductor-gwastools</requirement>
+        <requirement type="package" version="1.6.4">r-optparse</requirement>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+        Rscript '$__tool_directory__/manhattan.R'
+        --file ${file}
```
Comment (bgruening) [inline tools/gwastoos/manhattan_plot.xml]: quote the data param here

File: tools/gwastoos/manhattan_plot.xml [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+        #if $thinselect.thinning == 'yes':
+            --thin ${thinselect.thin}
+            #if $thinselect.ppb:
+                --ppb ${thinselect.ppb}
+            #end if
+        #end if
+    ]]></command>
+    <inputs>
+        <param name="file" type="data" format="tsv, tabular" label="GWAS data file"/>
+        <param name="chromosome" type="integer" value="2" label="Chromosome column number"/>
```
Comment (bgruening) [inline tools/gwastoos/manhattan_plot.xml]: use a data_column param ...

File: tools/gwastoos/.shed.yml (hunk 3 above)
Comment (astrovsky01) [reply tools/gwastoos/.shed.yml]: Yes, it should be

File: tools/gwastools/manhattan_plot.xml [hunk 7]
```diff
@@ -0,0 +1,88 @@
+<tool id="manhattan.plot" name="Manhattan plot" version="0.1.0" python_template_version="3.5">
+    <description>Generate plots for use with GWAS pipelines</description>
+    <requirements>
+        <requirement type="package" version="3.6">r-base</requirement>
+        <requirement type="package" version="1.30.0">bioconductor-gwastools</requirement>
+        <requirement type="package" version="1.6.4">r-optparse</requirement>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
```
Comment (pvanheus) [inline tools/gwastools/manhattan_plot.xml]: consider `detect_errors="aggressive"`

**Pull request merged**